src/logging.h

```diff
@@ -104,13 +104,15 @@ namespace BCLog {
         std::atomic<Level> m_log_level{DEFAULT_LOG_LEVEL};
 
         /** Log categories bitfield. */
-        std::atomic<uint32_t> m_categories{0};
+        std::atomic<uint32_t> m_categories{BCLog::NONE};
```

Comment:
In commit "logging: minor encapsulation improvement and use BCLog::NONE instead of 0" (1fae74149507e5914a7bb7986e4906053b8dbbbb)

Would be good to add "refactor:" prefix to commit message so it is obvious this is not supposed to change behavior.

Reply:
Done.